administrator/language/de-DE/plg_system_jooa11y.ini

```
@@ -152,6 +152,8 @@ PLG_SYSTEM_JOOA11Y_QA_BAD_ITALICS="Fett- und Kursiv-Tags haben eine semantische
 PLG_SYSTEM_JOOA11Y_QA_BAD_LINK="Schlechter Link gefunden. Link scheint auf eine Entwicklungsumgebung zu verweisen. Dieser Link verweist auf: <strong>%(el)</strong>"
 PLG_SYSTEM_JOOA11Y_QA_BLOCKQUOTE_MESSAGE="Ist dies eine Überschrift? <strong>%(bqHeadingText)</strong>"
 PLG_SYSTEM_JOOA11Y_QA_BLOCKQUOTE_MESSAGE_TIP="Blockquotes sollten nur für Zitate verwendet werden. Wenn dies eine Überschrift sein soll, dieses Blockquote in eine semantische Überschrift (z.&#8239;B. Überschrift 2 oder Überschrift 3) ändern."
+PLG_SYSTEM_JOOA11Y_QA_DUPLICATE_ID="<strong>Doppelte ID</strong> gefunden. Doppelte ID Fehler sind dafür bekannt, Probleme mit Hilfstechnologien zu verursachen, wenn diese versuchen den Inhalt zu verarbeiten."
+PLG_SYSTEM_JOOA11Y_QA_DUPLICATE_ID_TIP="Bitte lösche oder passe die folgende ID an: <strong>%(id)</strong>"
```

```suggestion
PLG_SYSTEM_JOOA11Y_QA_DUPLICATE_ID_TIP="Bitte die folgende ID löschen oder anpassen: <strong>%(id)</strong>"
```

Neutrale Sprache verwenden.